src/main/java/fun/epak/pak/utility/FileUploadUtil.java

```diff
@@ -0,0 +1,28 @@
+package fun.epak.pak.utility;
+
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+public class FileUploadUtil {
+    public static void saveFile(String uploadDir, String fileName,
+                                MultipartFile multipartFile) throws IOException {
+        Path uploadPath = Paths.get(uploadDir);
+
+        if (!Files.exists(uploadPath)) {
+            Files.createDirectories(uploadPath);
+        }
+
+        try (InputStream inputStream = multipartFile.getInputStream()) {
+            Path filePath = uploadPath.resolve(fileName);
+            Files.copy(inputStream, filePath, StandardCopyOption.REPLACE_EXISTING);
+        } catch (IOException ioe) {
+            throw new IOException("Could not save image file: " + fileName, ioe);
```

Comment:
To nie ma sensu. Jeśli już rzucasz IOException to lepiej rzucić ten orginalny, bo teraz gubimy stack trace. Idealnie by było zrobić tak jak napisałem przy kontrolerze